app/_hub/kong-inc/aws-lambda/index.md

```diff
@@ -128,7 +133,26 @@ params:
       default: "`false`"
       description: |
         An optional value that defines whether the response format to receive from the Lambda to [this format](https://docs.aws.amazon.com/apigateway/latest/developerguide/set-up-lambda-proxy-integrations.html#api-gateway-simple-proxy-for-lambda-output-format). Note that the parameter `isBase64Encoded` is not implemented.
-
+    - name: awsgateway_compatible
+      required: false
+      default: "`false`"
+      description: |
+        An optional value that defines whether the plugin should wrap requests into the Amazon API gateway.
+    - name: proxy_url
+      required: semi
+      default:
+      description: |
+        An optional value that defines whether the plugin should connect through the given proxy server URL. This value is required if `proxy_scheme` is defined.
+    - name: proxy_scheme
+      required: semi
+      default:
+      description: |
+        An optional value that defines which HTTP protocol scheme to use in order to connect through the proxy server. The schemes supported are: `http` and `https`. This value is required if `proxy_url` is defined.
+    - name: skip_large_bodies
+      required: false
+      default: "`true`"
+      description: |
+        An optional value that defines whether very large bodies (that are buffered to disk) should be sent by Kong. Note that sending very large bodies will have an impact on the system memory.
```

Comment:
Since "very large" is ambiguous, can we provide the actual size for the threshold?

Reply:
that is deteremined by the Nginx config directive that sets the value of when to start caching to disk. Don't have it handy....

Reply:
@Tieske would that be the directive you're referring to? http://nginx.org/en/docs/http/ngx_http_core_module.html#client_body_buffer_size

Reply:
@Tieske specifically: https://docs.konghq.com/1.3.x/configuration/#client_body_buffer_size

So maybe changing the above description to

"An optional value that defines whether very large bodies (that are buffered to disk) should be sent by Kong. The actual size for the threshold is defined via the [client_body_buffer_size](https://docs.konghq.com/latest/configuration/#client_body_buffer_size) configuration property. Note that sending very large bodies will have an impact on the system memory."

will suffice?

Reply:
```suggestion
        An optional value that defines whether Kong should send large bodies that are buffered to disk. To define the threshold for the body size, use [client_body_buffer_size](https://docs.konghq.com/latest/configuration/#client_body_buffer_size) property. Note that sending large bodies will have an impact on system memory.
```

Reply:
@murillopaula , I committed the suggestion. @Tieske may issue a PR if he wants to, it's GTG as is.